### PR TITLE
[Fix] Docs update

### PR DESCRIPTION
**Pull request by makylfang**
# PR

## Describe your changes

- Updated docs with `reconnect()` usage

## Checklist before requesting a review

- [x] You have performed a self-review of your own code
- [x] You are using approved terminology
- [x] Your changes have been tested locally
- [x] Your code builds clean without any errors or warnings

### CODE REVIEW

**Review by shaloo — CHANGES_REQUESTED** [review]
No value add wrt usage as this info is already there in https://dev-authsdk-ref-guide.netlify.app/classes/authprovider#reconnect and https://dev-authsdk-ref-guide.netlify.app/classes/authprovider#canReconnect.

Can we add one or two lines to guide the dev when/how to use? For e.g.

Auth allows devs to determine if a user is already logged in via another tab/browser-window and automatically reconnect. Before reconnecting a user (when a user action results in closing active tab / window, refreshing the page or reopening a new window while still logged in) make sure you first check using `canConnect()`.

Side Effects:  Initial state is same but same wallet can show diff networks if user modifies in one window and then refers to the previous connected one.

Reason: IsLoggedIn and CanReconnect are similar though meant for diff purposes hence a bit more clarity is helpful for new devs.

**Comment by makylfang** [comment]
> No value add wrt usage as this info is already there in https://dev-authsdk-ref-guide.netlify.app/classes/authprovider#reconnect and https://dev-authsdk-ref-guide.netlify.app/classes/authprovider#canReconnect.
> 
> Can we add one or two lines to guide the dev when/how to use? For e.g.
> 
> Auth allows devs to determine if a user is already logged in via another tab/browser-window and automatically reconnect. Before reconnecting a user (when a user action results in closing active tab / window, refreshing the page or reopening a new window while still logged in) make sure you first check using `canConnect()`.
> 
> Side Effects: Initial state is same but same wallet can show diff networks if user modifies in one window and then refers to the previous connected one.
> 
> Reason: IsLoggedIn and CanReconnect are similar though meant for diff purposes hence a bit more clarity is helpful for new devs.

I think devs can figure out it out.

**Review by shaloo — APPROVED** [review]
ok, if you say so :-)

File: usage.md [hunk 1]
```diff
@@ -149,6 +149,14 @@ Check if a user is logged in
 const isloggedIn = await auth.isLoggedIn() // boolean
 ```
 
+Check and reconnect if possible
```
Comment (shaloo) [inline usage.md]:
```suggestion
Check and reconnect, if required, within a 30-minute window after logout.

**Pull request merged**